### DISCUSSION TITLE
Caddy config?

### DISCUSSION THREAD

**Discussion by Horgul123** [Donkie/Spoolman#223] [Q&A] [answered]
I saw in the FAQ that I need to enable https to get the QR code working.
I tried to set up Caddy, but it's not working from outside the server (in a commandline https seems to work)

Can someone provide a Caddyfile which allows to use the QR function? Spoolman is running on a local server without an official domain

**Comment by Horgul123** [comment, marked as answer]
Got an answer from the Caddy community.
The Caddyfile should look like this:

https:// {
	tls internal {
		on_demand
	}

	reverse_proxy localhost:7912
}

**Comment by Fribb** [comment]
Thank you, searched the last 5 hours for how to get this to work.

Since I used Unraid to host the Caddy and Spoolman Docker containers I had to change the config to the following

```
https:// {
	tls internal {
		on_demand
	}

	reverse_proxy <server IP>:7912
}
```

Change the `<server IP>` to the, well, IP of the server and restart the Caddy container.
